comparators/impossible_angles.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+const impossibleAngle = require('../lib/impossible_angle');
+
+module.exports = impossibleAngleMotorableHighways;
+/*
+  check impossible angles in highways
+*/
+function impossibleAngleMotorableHighways(newVersion) {
```

Comment:
oldVersion can be taken as an imput parameter since it's used in `motorable highway method`